test/test.js

```diff
@@ -1,51 +1,81 @@
-import test from 'ava'
+'use strict';
 
-import flyPug from '../'
+const join = require('path').join;
+const test = require('tape').test;
+const Fly = require('fly');
 
-const fly = {
-  filter (name, plugin) {
-    this[name] = plugin
+const dir = join(__dirname, 'fixtures');
+const tmp = join(__dirname, 'tmp');
+
+const want = `var module = (function () {\n'use strict';\n\nfunction a () {\n  return 'a';\n}\n\nfunction b () {\n  return 'b';\n}\n\nfunction entry () {\n  a();\n  b();\n}\n\nreturn entry;\n\n}());`;
+
+const getOps = () => ({
+  rollup: {
+    plugins: [require('rollup-plugin-babel')()]
+  },
+  bundle: {
+    sourceMap: false,
+    moduleName: 'module',
+    format: 'iife'
   }
-}
+});
+
+const fly = new Fly({
+  plugins: [{func: require('../')}],
+  tasks: {
+    a: function * (o) {
+      const t = o.val;
 
-flyPug.call(fly)
+      const opts = getOps();
+      yield this.source(`${dir}/entry.js`).rollup(opts).target(tmp);
 
-test('bundle', async t => {
-  const data = ''
-  const options = {
-    file: {
-      base: 'entry.js',
-      dir: 'fixtures'
+      const res = yield this.$.read(`${tmp}/entry.js`, 'utf8');
```

Comment:
Will this kind of interface be documented in new API Docs ? It could help a lot when writing tests for new plugins.